Practice/Practice.Repository/HumanResourcesRepository.cs

```diff
@@ -77,6 +57,95 @@ public async Task<ViewModels.Employee> GetEmployeeById(int id)
+                        new BusinessEntityAddress
+                        {
+                            Address = new Address
+                            {
+                                AddressLine1 = employee.Address,
+                                AddressLine2 = Guid.NewGuid().ToString(), // to avoid bug with 2 different employees having the same address
+                                City = employee.City,
+                                PostalCode = employee.PostalCode,
+                                rowguid = Guid.NewGuid(),
+                                ModifiedDate = DateTime.Now,
+                                StateProvince = await context.StateProvince.FirstOrDefaultAsync()
+                            },
+                            AddressType = await context.AddressType.FirstOrDefaultAsync(at => at.Name == "Home"),
+                            ModifiedDate = DateTime.Now,
+                            rowguid = Guid.NewGuid()
+                        }
+                    }
+                },
+                FirstName = employee.FirstName,
+                LastName = employee.LastName,
+                PersonType = "EM",
+                NameStyle = false,
+                EmailPromotion = 0,
+                rowguid = Guid.NewGuid(),
+                ModifiedDate = DateTime.Now,
+                PersonPhone = new List<PersonPhone> {
+                    new PersonPhone
+                    {
+                        PhoneNumberType = await context.PhoneNumberType.FirstOrDefaultAsync(pnt => pnt.Name == "Cell"),
+                        ModifiedDate = DateTime.Now,
+                        PhoneNumber = employee.PhoneNumber
+                    }
+                }
+            };
+
+            // Create new employee with random properties
+            Random random = new Random();
+            Employee newEmployee = new Employee
+            {
+                Person = person,
+                NationalIDNumber = random.Next().ToString(),
+                LoginID = $"adventure-works/{employee.FirstName.ToLower() + random.Next().ToString()}",
+                JobTitle = "Intern",
+                BirthDate = new DateTime(random.Next(1970, 2000), random.Next(1, 12), random.Next(1, 28)),
+                MaritalStatus = "S",
+                Gender = random.Next(0, 1) == 1 ? "M" : "F",
+                HireDate = DateTime.Now - TimeSpan.FromDays(1),
+                SalariedFlag = false,
+                VacationHours = 25,
+                SickLeaveHours = 0,
+                CurrentFlag = true,
+                rowguid = Guid.NewGuid(),
+                ModifiedDate = DateTime.Now
+            };
+
+            context.Employee.Add(newEmployee);
+            await context.SaveChangesAsync();
+        }
+
+        public async Task DeleteEmployee(int id)
+        {
+            if (id == 0)
+                return;
+
+            Employee employee = await context.Employee.SingleAsync(x => x.BusinessEntityID == id);
```

Comment:
używajmy var tam gdzie to możliwe: [doc](https://docs.microsoft.com/pl-pl/dotnet/csharp/language-reference/keywords/var)